docs/public/static/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "MUI",
   "short_name": "MUI",
-  "display": "MUI",
+  "display": "minimal-ui",
```

Comment:
I don't know how this is used, but what does minimal-ui even reference too?

Reply:
cc @siriwatknp looks like we broke something with this change

Reply:
https://github.com/mui-org/material-ui/pull/28355/files#r709135180